src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml

```diff
@@ -57,99 +57,90 @@
-                                            </p>
-                                        }
-                                        <p class="govuk-body">
-                                            <span class="govuk-!-font-weight-bold">Found on:</span>
-                                            @if (result.RoEPAOApproved)
-                                            {
-                                                <span>RoEPAO</span>
-                                            }
-                                            @if (result.RoEPAOApproved && result.RoATPApproved)
-                                            {
-                                                <span> and </span>
-                                            }
-                                            @if (result.RoATPApproved)
-                                            {
-                                                <span>RoATP</span>
-                                            }
-                                            @if (!result.RoEPAOApproved && !result.RoATPApproved)
-                                            {
-                                                switch (result.EasApiOrganisationType)
-                                                {
-                                                    case "Charity":<span>Charity Commission</span> break;
-                                                    case "Company": <span>Companies House</span> break;
-                                                    case "PublicSector": <span>EASAPI - Public Sector</span>break;
-                                                    case "EducationOrganisation": <span>EASAPI - Education Organisation</span> break;
-                                                    default: <span>EASAPI</span>break;
-                                                }
-                                            }
-                                        </p>
-                                    }
-
-                                    @if (result.RoEPAOApproved && result.OrganisationIsLive)
-                                    {
-                                        <p class="govuk-body">
-                                            <strong class="govuk-tag">
-                                                ACTIVE ACCOUNT
-                                            </strong>
-                                        </p>
-                                    }
-                                </td>
-                            </tr>
+                            @("Address not found");
+                        }
+                        else
+                        {
+                            var addressArray = new[] { @result.Address?.Address1, @result.Address?.City, @result.Address.Postcode };
+                            @string.Join(", ", addressArray.Where(s => !string.IsNullOrEmpty(s)));
+                        }
```

Comment:
I don't see how this would have ever worked. The return value of `string.Join()` is not being used!

I didn't add this code, despite GitHub's green highlighting.

Reply:
Doesn't it just drop the value onto the screen as output?

Reply:
Ah yes, this is razor